src/cfnlint/rules/resources/cloudfront/Aliases.py

```diff
@@ -33,14 +33,12 @@ def match(self, cfn):
                     if isinstance(alias, str) and alias not in FUNCTIONS:
                         wildcard = alias.split('.')
                         if '*' in wildcard[1:]:
-                            message = 'Invalid use of wildcards: {}'.format(alias)
                             path = result['Path'] + ['Aliases']
-                            matches.append(
-                                RuleMatch(path, message.format(('/'.join(result['Path'])))))
+                            message = 'Invalid use of wildcards: {} at {}'.format(alias, path)
```

Comment:
```suggestion
                            message = 'Invalid use of wildcards: {} at {}'.format(alias, '/'.join(result['Path']))
```

Reply:
I think we need the path to be joined here.

Reply:
Yeah, sorry. Fixed it now.